install.py

```diff
@@ -1,30 +1,22 @@
 import os
-
 import launch
 
-try:
-    import trt_paths
-except Exception as e:
-    print("Could not find TensorRT directory; skipping install", e)
-
-
 def install():
     if not launch.is_installed("tensorrt"):
-        trt_whl_path = os.path.join(trt_paths.trt_path, "python")
-        matching_files = [os.path.join(trt_whl_path, x) for x in os.listdir(trt_whl_path)]
-        matching_files = [x for x in matching_files if "tensorrt-" in x and "cp310" in x]
-        if len(matching_files) == 0:
-            print(f"Could not find TensorRT .whl installer; looked in {trt_whl_path}")
-
-        whl = matching_files[0]
-        launch.run_pip(f'install "{whl}"', "TensorRT wheel")
-
-    if not launch.is_installed("pycuda"):
-        launch.run_pip(f'install pycuda', "pycuda")
+        launch.run_pip(f"install nvidia-cudnn-cu11==8.9.4.25", "nvidia-cudnn-cu11")
+        launch.run_pip(f"install --pre --extra-index-url https://pypi.nvidia.com tensorrt==9.0.0.post11.dev1", "tensorrt")
+        launch.run(['python','-m','pip','uninstall','-y','nvidia-cudnn-cu11'], "removing nvidia-cudnn-cu11")

```

Comment:
Can we add a comment on why this is done ? And probably also state that this is subject to change :D